acceptance/api/v1/configurations_mutation_test.go

```diff
@@ -554,6 +554,10 @@ var _ = Describe("Configurations API Application Endpoints, Mutations", func() {
 		})
 
 		It("returns a 'not found' when the application does not exist", func() {
+			// This requires a valid configuration
+			configuration := catalog.NewConfigurationName()
```

Comment:
This is already called in the `BeforeEach` so it's not needed here (?)

Reply:
Done. 60a1964edefdd9b1e343980508f61cfd3067a1f7